tools/table-server/src/main/resources/org/jmeterplugins/protocol/http/control/simple-table-server.cmd

```diff
@@ -27,9 +27,7 @@ setlocal
 
 cd /D %~dp0
 
-set CP=..\lib\ext\ApacheJMeter_core.jar;..\lib\jorphan.jar;..\lib\ext\jmeter-plugins-table-server-2.0.jar
+set CP=..\lib\ext\ApacheJMeter_core.jar;..\lib\jorphan.jar;..\lib\ext\jmeter-plugins-table-server-2.1.jar
```

Comment:
this will always cause the problem with forgetting to change the line. Right way is to enable resources filtering in Maven and put here the properties from Maven to be automatically resolved by mvn.